src/navier_stokes/INSStaggeredHierarchyIntegrator.cpp

```diff
@@ -925,11 +919,10 @@ INSStaggeredHierarchyIntegrator::initializeHierarchyIntegrator(Pointer<PatchHier
         if (d_output_U)
         {
             d_visit_writer->registerPlotQuantity("U", "VECTOR", d_U_cc_idx, 0, d_U_scale);
-            for (unsigned int d = 0; d < NDIM; ++d)
+            for (unsigned int i = 0; i < NDIM; ++i)
```

Comment:
I think we usually use `d` for loops over dimensions so I would rather keep it that way.

Reply:
FWIW, I am thinking here in terms of vector/tensor indices (i,j).

Reply:
I have made this a bit more explicit. Let me know if the current state looks OK.